api/can_api_definition.py

```diff
@@ -45,6 +45,12 @@ class _Actuals(pydantic.BaseModel):
     cumulativeConfirmedCases: int = pydantic.Field(
         ..., description="Number of confirmed cases so far"
     )
+    cumulativePosTested: Optional[int] = pydantic.Field(
```

Comment:
I don't think there's a reason to abbreviate here - doesn't look like we do anywhere else

Reply:
I can go either way